app/Console/Commands/AddExpression.php

```diff
@@ -94,16 +93,16 @@ public function handle()
 
                 return 1;
             }
-            $file = explode('.', $key)[0];
-            $expression_key = explode('.', $key)[1];
+
+            [$file, $expression_key] = explode('.', $key);
 
             foreach (['en', 'hu'] as $language) {
                 $path = '/resources/lang/'.$language.'/'.$file.'.php';
                 if (! file_exists(base_path($path))) {
                     if (! $this->confirm('Do you want to create a new '.($language == 'en' ? 'english' : 'hungarian').' file named '.$file.'.php?')) {
                         $this->info('Action cancelled.');
 
-                        return;
+                        return 1;
```

Comment:
For example here

Reply:
But I didn't delete those return 1s, I'm the one who added them since those empty returns previously there returned 0 on failure :smile:

Reply:
Oh, i`m blind :D these lines does not mean necessary that the process failed, just interrupted by a user. But it's okay returning 1 as well.